persist/file.go

```diff
@@ -0,0 +1,122 @@
+package persist
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/cloudflare/redoctober/config"
+	"github.com/cloudflare/redoctober/keycache"
+	"github.com/cloudflare/redoctober/passvault"
+)
+
+// File implements a file-backed persistence store.
+type File struct {
+	config *config.Delegations
+	cache  *keycache.Cache
+	state  string
+	blob   []byte
+}
+
+// Valid ensures the configuration is valid for a file store. Note
+// that it won't validate the policy, it will just ensure that one
+// is present.
```

Comment:
This comment contradicts itself. I am sufficiently amused that I felt it was worth pointing out.